### PR TITLE
Internalize keywords for Gas Lift Optimization

### PR DESCRIPTION
**Pull request by joakim-hove**
This PR internalizes the kewyords `LIFTOPT, GLIFTOPT` and `WLIFTOPT` which controls the use of gas lift optimization.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
This fails to link in serialization code in opm-simulators; my blood sugar is a bit low too see what is missing now - any hints appreciated :-)

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/parser/eclipse/EclipseState/Schedule/GasLiftOpt.hpp [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+    bool operator==(const GasLiftOpt& other) const;
+
+    template<class Serializer>
+    void serializeOp(Serializer& serializer)
+    {
+        serializer(m_increment);
+        serializer(m_min_eco_gradient);
+        serializer(m_min_wait);
+        serializer(m_all_newton);
+        serializer(m_groups);
```
Comment (akva2) [inline opm/parser/eclipse/EclipseState/Schedule/GasLiftOpt.hpp:236]: serializer.map(m_groups);

File: opm/parser/eclipse/EclipseState/Schedule/GasLiftOpt.hpp [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+        serializer(m_wells);
```
Comment (akva2) [inline opm/parser/eclipse/EclipseState/Schedule/GasLiftOpt.hpp:237]: serializer.map(m_wells);

File: opm/parser/eclipse/EclipseState/Schedule/GasLiftOpt.hpp (hunk 1 above)
Comment (joakim-hove) [reply opm/parser/eclipse/EclipseState/Schedule/GasLiftOpt.hpp]: Thank you!

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
I'll let jenkins do another round on this and then I intend to merge it.

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**